src/binding/bindingAttributeSyntax.js

```diff
@@ -255,9 +328,7 @@
                 bindingApplied = true;
             }
 
-            if (bindingApplied) {
```

Comment:
It looks like `bindingApplied` is no longer used and can be removed from this context.

Reply:
Thanks.